client/src/app/site/assignments/components/assignment-poll/assignment-poll.component.html

```diff
@@ -111,6 +111,7 @@ <h3 translate>Ballot</h3>
                     <span>{{ key | translate }}</span>:
                 </div>
                 <div>
+                    {{ poll[key] | precisionPipe }}
                     {{ pollService.getSpecialLabel(poll[key]) }}
```

Comment:
Yes, I didnt want to do this, because I tried to avoid merge conflicts. This was just an example (and for this reason the note how to include the pipe into a service)